src/TLuaInterpreter.cpp

```diff
@@ -5182,6 +5182,23 @@ int TLuaInterpreter::getMudletHomeDir( lua_State * L )
     return 1;
 }
 
+// returns search paths for LuaGlobal itself to look at when loading other modules
+// follows the principle of closest paths to the binary first, furthest away lasy
+int TLuaInterpreter::getMudletLuaDefaultPaths( lua_State * L )
+{
+    lua_newtable( L );
+    lua_createtable(L,2,0);
+    // add filepath relative to the binary itself (one usecase is AppImage on Linux)
+    QString nativePath = QDir::toNativeSeparators( QCoreApplication::applicationDirPath() + "/mudlet-lua/lua/" );
```

Comment:
Oh, it returns a table! I was somehow reading it as returning a single string (mostly in connection with your changes to LuaGlobal.cs